src/components/layout/NavBar.jsx

```diff
@@ -52,19 +52,19 @@ const NavLogo = styled.div`
 // TODO: due to issue https://github.com/styled-components/styled-components/issues/439,
 // bgcolor prop triggers react warning now, need to fix
 const NavItem = styled(Link)`
-  border-left: 1px solid #9b9b9b;
+  border-left: 2px solid #f4f4f4;
```

Comment:
why did the color change on the border? think any change of design need to be discussed with @uchicagovivi @zflamig

Reply:
sorry... my bad...
That color is #D1D1D1, and please name it “smoke"
I'm updating it on style guide too. thanks!